### ISSUE TITLE
ColorPicker - the button label is not announced by screen reader

### ISSUE THREAD

**Issue by kmmkwong** [salesforce/design-system-react#3094]
We have an accessibility defect logged on ColorPicker where the button label is not announced by screen reader.  Instead, the hex number for the selected color is been read.  When assertive text is provided for "assertText" prop, it is not read neither.

**Comment by kmmkwong** [comment]
Fixing it here - https://github.com/salesforce/design-system-react/pull/3095. Not sure how to link
